pkg/controllers/resources/endpoints/syncer.go

```diff
@@ -28,6 +29,7 @@ func New(ctx *synccontext.RegisterContext) (syncertypes.Object, error) {
 
 	return &endpointsSyncer{
 		GenericTranslator: translator.NewGenericTranslator(ctx, "endpoints", &corev1.Endpoints{}, mapper),
+		Importer:          pro.NewImporter(mapper),
```

Comment:
Can this cause issues when enabled and we sync services that create automatically endpoints? I think this will be problematic and probably requires testing or we remove for now

Reply:
Importer also doesn't make sense if the SyncToVirtual function does not create the object